### PR TITLE
Add support for 3-arg subtraction

### PR DESCRIPTION
**Pull request by mike-north**
We promised our customers that this would be possible!

```py
sub(3, 4, 2) # -3
```

Apparently we never finished this, so here's the fix

### CODE REVIEW

**Comment by mike-north** [comment]
Hello class! This is a comment

File: src/calc.py [hunk 1]
```diff
@@ -9,12 +9,13 @@ def add(a, b):
     """
     return a + b
 
-def sub(a, b):
+def sub(a, b, c = 0):
```
Comment (mike-north) [inline src/calc.py:12]: I don't like the argument name `c`. Can we use something more descriptive?
```suggestion
def sub(a, b, third_operand = 0):
```

File: src/calc.py [hunk 2]
```diff
@@ -9,12 +9,13 @@ def add(a, b):
...
     """
     Subtract some numbers
 
     ```py
     sub(8, 3) # 5 
+    sub(8, 3, 2) # 3
     ```
     """
-    return a - b
+    return a - b - c
```
Comment (mike-north) [inline src/calc.py:21]:
```suggestion
    return a - b - third_operand
```

**Review by mike-north — COMMENTED** [review]
Looks great! Clearly you know your ABCs better than subtraction

**Review by kevnshin — APPROVED** [review]
lgtm

**Comment by wookee55** [comment]
ok

**Review by m1san — APPROVED** [review]
LGTM! 🚀

**Pull request merged**